src/main/resources/en.yml

```diff
@@ -241,3 +241,9 @@ en:
+       measurements: ["teaspoon", "tablespoon", "cup", "pint", "quart", "gallon"]
```

Comment:
Fair enough. I just thought I'd bring it up since so few people use the imperial measurement system :)